flask_lesson_advanced/blueprint/product_main.py

```diff
@@ -0,0 +1,54 @@
+from flask import Blueprint, render_template, request, session
+import os
+from werkzeug.utils import secure_filename
+from infractructure import DB
+from models import Product_class
+
+product = Blueprint('product', __name__, template_folder='template_product',
+                    static_folder='static_product')
+
+
+@product.route('/product', methods=['GET'])
+def products_fun():
+    arg_price = request.args.get("price")
+    products_list = [{"name": rest.name, "id": rest.id, "price": rest.price,
+                      "img_name": rest.img_name,
+                      "description": rest.description}
+                     for rest in DB['products']]
```

Comment:
move this on if statement bellow

Reply:
we had chat about it and it no need to be changed